backend/corpora/common/providers/crossref_provider.py

```diff
@@ -32,7 +32,10 @@ def __init__(self) -> None:
     @staticmethod
     def parse_date_parts(obj):
         date_parts = obj["date-parts"][0]
-        return (date_parts[0], date_parts[1], date_parts[2])
+        year = date_parts[0]
+        month = date_parts[1] if len(date_parts) > 1 else 1
+        day = date_parts[2] if len(date_parts) > 2 else 1
+        return (year, month, day)
```

Comment:
Will this set the month to jan and the date to the 1st if that information isnt included? Is that the desired behavior? Why?